Enterprise-CMCS/macpro-mdct-mcr
# Review PDF Button Change

### Description

After a form in MLR  or MCPAR is submitted, in the Review & Submit page, the button to open PDF should look like this:

<img width="215" alt="Screenshot 2023-06-15 at 2 37 16 PM" src="https://github.com/Enterprise-CMCS/macpro-mdct-mcr/assets/127151429/3340a0f5-27fc-4e63-922b-a1da7a8e6a64">


### Related ticket(s)


MDCT-2564

---
### How to test

1) Login to MLR 
2) Click the Review & Submit button from the side menu
3) Look at the Review PDF button and confirm that it's white bg with blue text
4) Fill out MLR form and submit
5) Go back to Review & Submit  page
6) Review PDF button should look like the image above, blue bg with white text

### Important updates


N/A

---
### Author checklist


---


_convert to a different template: [test → val](?expand=1&template=test-to-val-deployment.md)_ | _[val → prod](?expand=1&template=val-to-prod-deployment.md)_
